### ISSUE TITLE
FR: Anonymous File Pushes

### ISSUE THREAD

**Issue by pglombardo** [pglombardo/PasswordPusher#1046]
## 🚀 Feature Request

From the Feedback form:

> We have deployed your tool on our local infrastructure, however we encountered an issue that login is mandatory in order to push files via pwpush website, I have a question is there a way to by pass this? As I’m unable to find it in documentation and it says that it is mandatory in order to work, however in our configuration we would like to have an option to push files in pwpush without requiring a login.

**Comment by elghazal-a** [comment]
Yeah I agree, it would be great to have the possibility to share files also without login requirements. We self-host pwpush and it's used only internally and we don't require login for text sharing, why should we require it specifically for images?

BTW, thanks for your great work

**Comment by Indigo744** [comment]
We would also be interested in this feature.

@pglombardo What would be needed for this? I have never worked in ruby but I'm willing to give it a try and get the ball rolling 😸
